packages/mui-material/src/Slider/Slider.d.ts

```diff
@@ -64,6 +64,8 @@ type SliderThumbProps = NonNullable<SliderTypeMap['props']['componentsProps']>['
 type SliderValueLabelProps = NonNullable<SliderTypeMap['props']['componentsProps']>['valueLabel'];
 type SliderInputProps = NonNullable<SliderTypeMap['props']['componentsProps']>['input'];
 
+export { SliderValueLabelProps } from '@mui/base/SliderUnstyled';
```

Comment:
Ideally I would have done:

```suggestion
export type SliderValueLabelProps = NonNullable<SliderTypeMap['props']['componentsProps']>['valueLabel'];
type SliderInputProps = NonNullable<SliderTypeMap['props']['componentsProps']>['input'];
```

But for some reason, there is a union type with `undefined` that make it fail.